README.md

```diff
@@ -96,7 +96,8 @@ $ source venv/bin/activate
 
 ```bash
 # Must be in activated virtual environment
-$ pytest tests/test_wave_01.py
+$ 
+
```

Comment:
Using `git diff` before you commit is a good way to catch these sorts of accidental changes.